hw/top_earlgrey/cdc/cdc_waivers.tcl

```diff
@@ -50,6 +50,131 @@ set_rule_status -rule {W_GLITCH} -status {Waived}          \
     (GlitchOutput=~"*u_sync_1*")} \
   -comment {Waive PADs input goes into synchronizer}
 
+# Driving from PAD RESET to SPI TPM
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"IO*") && (ReceivingFlop=~"*u_spi_device.u_spi_tpm.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"SPI_DEV_CS_L*") && (ReceivingFlop=~"*u_spi_device.u_spi_tpm.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
+  -expression {(DrivingSignal=~"top_earlgrey.u_pinmux_aon*") && (ReceivingFlop=~"*u_spi_device.u_spi_tpm.*")} \
+  -comment {PAD RESET driving to SPI TPM}
+
+set_rule_status -rule {W_ASYNC_RST_FLOPS} -status {Waived}                           \
```

Comment:
hmm... this one doesn't look like it's from PAD.
Is it bad that we are effectively waiving all resets to `spi`..?

Reply:
@eunchan should this be more specific? is that possible to do?

Reply:
These waivers were added in https://github.com/lowRISC/opentitan/pull/13739. I added spi_tpm to be more specific as requested. I will add more specific paths.

Reply:
it may not be easy to do, so just a suggestion. I think you guys are better judges of what makes the most sense here.

Reply:
As I re-checked the path, spi_tpm is the most detailed common path as follows.
![Screen Shot 2022-08-24 at 2 32 51 PM](https://user-images.githubusercontent.com/103145460/186527716-2316e108-af2a-4374-bd78-3ee8be34b012.png)
Otherwise, I can specify each detailed TPM register as above.
In addition, I will also add more detailed path on DrivingSignal, too.